src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/ChainPal.cs

```diff
@@ -49,7 +41,7 @@ internal sealed partial class ChainPal : IDisposable, IChainPal
             return status.dwError == 0;
         }
 
-        public IEnumerable<X509ChainElement> ChainElements
+        public X509ChainElement[] ChainElements
```

Comment:
> You mean @dotnet-bot, of course :smile:

Sure :smile: Wasn't trying to assign blame, just address the person most likely to follow-up on it :wink:

> I'd probably have done it as methods, but don't know that changing it is justified

I personally think it'd be worthwhile, but up to you. It's just too easy to accidentally use it incorrectly.